### PR TITLE
Anshuman resort level writeup

### PR DESCRIPTION
**Pull request by AnshumanNeon**
My writeup is complete. Actually it was complete long ago but i didn't open a pull request. You can merge it now. Tell me about the issues if there are any.

### CODE REVIEW

**Comment by Antibrain0** [comment]
This is looking pretty good! Some screenshots are out of order but that isn't really an issue. Some of the level appear to be cropped weirdly but that might just be my phone trying to "improve" readability.

As for the mess levels, I do like the idea of making them more linear, but as for removing wrapping back to the main room and things like that, that actually shouldn't be a big problem. We could just have that same level in the level metadata multiple times, with different level exits. (As in start exits to mess_1, end of mess_1 loads that same map data, but it exits to mess_2 in a different direction.)

Your writeup is great! Thank you for the help. I'll go ahead and review it, although I'll wait for cal to review to merge.

**Review by Antibrain0 — APPROVED** [review]
@Calverin , take a look at this when you can

**Review by Antibrain0 — APPROVED** [review]

File: design/design_resort.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Resort Write-Up
+
+This document features the writeup for the levels to be included in resort chapter of newleste.p8
+
+# Levels
+
+## Summary
+
+- The chapter starts with introductory levels that introduce to main mechanics.
```
Comment (Calverin) [inline design/design_resort.md:9]: that introduce the player to the main mechanics.

File: design/design_resort.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- Then the mess levels tests those mechanics.
```
Comment (Calverin) [inline design/design_resort.md:10]: The mess levels then test the player's knowledge of the mechanics.

File: design/design_resort.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+- The roof levels test the skill of player in those mechanics along with a boss fight.
```
Comment (Calverin) [inline design/design_resort.md:11]: The roof levels combine the previously introduced mechanics along with the added challenge of the Oshiro boss fight.

File: design/design_resort.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof06.png" alt="roof06" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof07.png" alt="roof07" />
+
+# Chapter Arc
+
+It should be said that first 4-6 levels are single screen ones and scrolling is gradually added later in the chapter. The most scrolling would be found in mess levels and roof levels.
+
+# Mess Levels Layout
+
+As the mess levels are intertwined, they are beyond the limits of pico-8 and the space limitations of newleste.p8.
```
Comment (Calverin) [inline design/design_resort.md:57]: As the mess levels in the base game are intertwined, they would be beyond the limitations we have for newleste.p8 in their original form.

File: design/design_resort.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+That is why the mess levels are given linear progression in newleste.p8 and the specific levels and their orders are debatable and subject to change.
```
Comment (Calverin) [inline design/design_resort.md:58]: Instead, we will have the mess levels be linear as with the rest of the levels in the game.

File: design/design_resort.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+A lot of new levels might have to be designed and a lot of given levels might have to be redesigned.
+
+Keeping in mind these constraints should the levels be made.
```
Comment (Calverin) [inline design/design_resort.md:61]: Keep this all in mind if you plan on (re)creating levels with mess blocks

File: design/design_resort.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+If there are demands to change a level or add new levels or change the order of levels they are to be discussed in the official Celeste Classic discord server.
```
Comment (Calverin) [inline design/design_resort.md:62]: This isn't really a necessary line imo

File: design/design_resort.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof01.png" alt="roof01" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof03.png" alt="roof03" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof04.png" alt="roof04" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof06.png" alt="roof06" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof07.png" alt="roof07" />
+
+# Chapter Arc
+
+It should be said that first 4-6 levels are single screen ones and scrolling is gradually added later in the chapter. The most scrolling would be found in mess levels and roof levels.
```
Comment (Calverin) [inline design/design_resort.md:53]: "introduced" instead of "added"

File: design/design_resort.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
```
Comment (Calverin) [inline design/design_resort.md:54]: Should probably discuss the Oshiro boss fight taking place in the roof levels here

**Review by Calverin — CHANGES_REQUESTED** [review]
Covered pretty much all needed ground with good level choices, nice job! I just made a few spelling/grammar suggestions and one requested inclusion in the Chapter Arc section

**Review by AnshumanNeon — COMMENTED** [review]
lets go with this, i'll commit to revert back to make sure wrong changes are not done

**Review by AnshumanNeon — COMMENTED** [review]
review

**Pull request closed without merging**